src/App.js

```diff
@@ -5,6 +5,7 @@ import Sound from "./Sound";
 import Search from "./Search";
 import auth, { init } from "./Auth";
 
+
```

Comment:
remove this space

Reply:
Just pushed new changes.

Reply:
I'll close the issue once these changes are reviewed and merged to master.